src/mrt.rs

```diff
@@ -84,9 +84,7 @@ where
     /// The `reload` argument must be smaller than or equal to [`MAX_VALUE`].
     ///
     /// [`MAX_VALUE`]: constant.MAX_VALUE.html
-    pub fn start(&mut self, reload: impl Into<Ticks>) {
-        let reload = reload.into();
-
+    pub fn start(&mut self, reload: Ticks) {
```

Comment:
I think diverging the inherent impls & the trait impls should be done very carefully & I can also imagine some weird error scenarios depending on which traits are in scope. In this case, it's probably acceptable (since there's nothing to `into` it from), but we have to be very careful.

Reply:
Good point, didn't think of that. Keep in mind though, that the current situation is only semi-permanent. At some point we'll probably retire support for embedded-hal 0.2.x, and the new methods have a `try_` prefix.

Reply:
Oh, and the whole point of the inherent methods is that they are already different: The inherent ones are infallible, while the embedded-hal one's are fallible. So the weird errors depending on traits in scope are already possible.

Overall, I think it's a minor issue, and one that will disappear in time.